src/modules/AutoinstConfig.rb

```diff
@@ -332,10 +334,9 @@ def update_profile_location(profile_location)
         profile_location = "usb:///#{DEFAULT_PROFILE_NAME}"
       elsif profile_location == "slp"
         profile_location = find_slp_autoyast
-        return if profile_location.nil?
+      else
+        profile_location
       end
-
-      profile_location
```

Comment:
If you're maintaining the variables in each if branch, I would no remove this line.